pyvista/examples/downloads.py

```diff
@@ -4933,6 +4933,48 @@ def download_dikhololo_night():  # pragma: no cover
     return texture
 
 
+def download_hydrogen_orbital(load=True):  # pragma: no cover
+    """Download a model of the electron cloud of the hydrogen atom.
```

Comment:
@adeak, please verify. My understanding is not at the level of a jumped up high school physics student.

Reply:
I've made one minor edit from "base state" to "ground state", otherwise this is perfect!

In the meantime I'll try to verify that this is plausibly a 3d orbital, just to be technically correct.